pandas/tests/io/json/test_json_table_schema.py

```diff
@@ -237,6 +237,23 @@ def test_build_series(self):
 
         assert result == expected
 
+    def test_read_json_from_to_json_results(self):
+        # GH 32383
+        df = pd.DataFrame(
+            {
+                "_id": {"0": 0},
+                "category": {"0": "Goods"},
+                "recommender_id": {"0": 3},
+                "recommender_name_jp": {"0": "浦田"},
+                "recommender_name_en": {"0": "Urata"},
+                "name_jp": {"0": "博多人形（松尾吉将まつお よしまさ）"},
+                "name_en": {"0": "Hakata Dolls Matsuo"},
+            }
+        )
+        result = pd.read_json(df.to_json())
+        df.index = df.index.astype(int)
```

Comment:
On initial glance it's confusing why this is required, but it's only because of how the DataFrame above is constructed. Alternately just use from_dict and specify the columns as the orient, like:

```python
df = pd.DataFrame({"_id": [0], "category": ["Goods"], ...}, orient="columns")
```

Reply:
I find a way to work around this, thanks